app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(16, 4) == [4, 4, 4, 4]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(13, 1) == [13]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    parts = split_integer(32, 5)
+    assert parts == sorted(parts)
```

Comment:
it's a bad test because `parts` is the result of `split_integer(32, 5)` and you're comparing `parts` to `parts`